### PR TITLE
New term entry for Java Math.sinh()

### LINKED ISSUE #914: [Term Entry] Java Math Methods: `.sinh()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.sinh()` Java method.

- In the `terms/` folder, make a new folder called `sinh`
- Inside the `sinh` folder, create a new markdown file called `sinh.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by shivanshugour45**
### Description

Adding sinh.md which explains the implementation of java.lang.Math.sinh().

Closes #914 

### Type of Change

- Adding a new entry

### Checklist

- [v] All writings are my own.
- [v] My entry follows the Codecademy Docs style guide.
- [v] My changes generate no new warnings.
- [v] I have performed a self-review of my own writing and code.
- [v] I have checked my entry and corrected any misspellings.
- [v] I have made corresponding changes to the documentation if needed.
- [v] I have confirmed my changes are not being pushed from my forked `main` branch.
- [v] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [v] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/sinh/sinh.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+Title: '.sinh()'
+Description: 'Returns the hyperbolic sine of a double-type value. The hyperbolic sine of x is defined to be (e^x - e^-x)/2 where e is Euler's number.'
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sinh/sinh.md:3]:
```suggestion
Description: 'Returns the hyperbolic sine of a double-type value.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@shivanshugour45, I did a first review and left a few comments.

**Comment by shivanshugour45** [comment]
Suggested changes are done.

**Review by SSwiniarski — APPROVED** [review]
@shivanshugour45 looks like were ready for review two.

File: content/java/concepts/math-methods/terms/sinh/sinh.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+The return type of the `.sinh()` method is a double.
+
+Some special cases for the `.sinh()` method include the following:
+
+- If the argument is `NaN`, then the result is `NaN`.
+- If the argument is infinite, then the result is an infinity with the same sign as the argument.
+- If the argument is zero, then the result is a zero with the same sign as the argument.
+
+The computed result will be within 2.5 ulps of the exact result.
```
Comment (HishamT) [inline content/java/concepts/math-methods/terms/sinh/sinh.md:32]: What does "ulps" mean?

File: content/java/concepts/math-methods/terms/sinh/sinh.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+The **`Math.sinh()`** method returns the hyperbolic sine of a double-type value.
+
+## Syntax
+
+```pseudo
+Math.sinh(double num)
+```
+
+The return type of the `.sinh()` method is a double.
```
Comment (HishamT) [inline content/java/concepts/math-methods/terms/sinh/sinh.md:24]: Types in Java should be surrounded by backticks.

**Review by HishamT — CHANGES_REQUESTED** [review]
Hey @shivanshugour45, I have finished your review. Left some comments for you. :grin:

File: content/java/concepts/math-methods/terms/sinh/sinh.md (hunk 2 above)
Comment (shivanshugour45) [reply content/java/concepts/math-methods/terms/sinh/sinh.md:32]: Added an explanation for ULP.

**Review by shivanshugour45 — COMMENTED** [review]
Suggested changes are done

File: content/java/concepts/math-methods/terms/sinh/sinh.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Some special cases for the `.sinh()` method include the following:
+
+- If the argument is `NaN`, then the result is `NaN`.
+- If the argument is infinite, then the result is an infinity with the same sign as the argument.
+- If the argument is zero, then the result is a zero with the same sign as the argument.
+
+The computed result will be within 2.5 ulps of the exact result.
+
+An ulp stands for unit of least precision. It calculates the distance between the given `double` or `float` value and the
+`double` or `float` value next larger in magnitude.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sinh/sinh.md:35]:
```suggestion
The computed result will be within 2.5 units of least precision (ulps) of the exact result.
```

The latter part of the text is more suitable for another entry (perhaps one about the `Math.ulp()` Java method).

File: content/java/concepts/math-methods/terms/sinh/sinh.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+Math.sinh(double num)
+```
+
+The return type of the `.sinh()` method is a `double`.
+
+Some special cases for the `.sinh()` method include the following:
+
+- If the argument is `NaN`, then the result is `NaN`.
+- If the argument is infinite, then the result is an infinity with the same sign as the argument.
+- If the argument is zero, then the result is a zero with the same sign as the argument.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sinh/sinh.md:30]:
```suggestion
- If `num` is `NaN`, then the result is `NaN`.
- If `num` is infinite, then infinity is returned with the same sign as `num`.
- If `num` is zero, then zero is returned with the same sign as `num`.
```

File: content/java/concepts/math-methods/terms/sinh/sinh.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.sinh()`** method returns the hyperbolic sine of a double-type value.
```
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sinh/sinh.md:16]:
```suggestion
The **`Math.sinh()`** method returns the hyperbolic sine of a `double`-type value.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@shivanshugour45 Finished with my review and left you some comments/suggestions. 😄

File: content/java/concepts/math-methods/terms/sinh/sinh.md (hunk 4 above)
Comment (shivanshugour45) [reply content/java/concepts/math-methods/terms/sinh/sinh.md:35]: Added the latter part as the description was asked by @HishamT.

**Comment by shivanshugour45** [comment]
@Dusch4593, suggested changes are done.

**Review by Dusch4593 — APPROVED** [review]
@shivanshugour45 LGTM! 👍🏻

What say you, @HishamT?

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by shivanshugour45** [comment]
Hi  @Dusch4593, This was my first contribution to open source community and it was a great experience. I'm not sure if I want to get credited or not but anyways what's the harm in that, let me know what I have to do. And Thank you for merging my PR looking forward to contributing more.😁

**Comment by Dusch4593** [comment]
@shivanshugour45 We definitely help you with getting credit! 🚀

You'll need to make sure that your GitHub username is attached to your Codecademy profile. You can do so by following these steps:

1. While logged on to codecademy.com, select your profile near the top-right.
2. Select "Profile".
3. Select "Edit Profile" (near the left-hand side).
4. Ensure your GitHub account handle is filled in the "GitHub Username" textbox (without the `@`):
Screen Shot 2022-10-21 at 11 23 32 AM
5. Scroll down and select "Save Changes".
6. Log out of your Codecademy Profile and back in through GitHub:
Screen Shot 2022-10-21 at 11 25 33 AM

And then you should be good! 😄 It might take a minute for the update to be reflected in the live entry page.

**Pull request merged**